spiffe.go

```diff
@@ -111,3 +112,33 @@ func FGetURINamesFromPEM(f io.Reader) (uris []string, err error) {
 	}
 	return uriNamesFromPEM(blob)
 }
+
+// GetURINamesFromExtensions retrieves URIs from the SAN extension of a slice of extensions
+func GetURINamesFromExtensions(extensions *[]pkix.Extension) (uris []string, err error) {
+	for _, ext := range *extensions {
+		if ext.Id.Equal(OidExtensionSubjectAltName) {
+			uris, err = getURINamesFromSANExtension(ext.Value)
+			if err != nil {
+				return
```

Comment:
Can we be explicit w/ the return vals here? We are in most other occasions, and I think it's easier to read that way anyways

Reply:
:+1: I will change it.